### PR TITLE
CCN3-1518: Display all applications on ARC search page

### PR DESCRIPTION
**Pull request by ricaduchateau**
## Description

Display all applications on ARC search page results table without having to search

## Todo's before PR

- [x] Rebase from release/v1.2.0
- [x] Unit tests passed (`make test`)
- [x] PR naming according our [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Commits-guideline)
- [x] PR description describes the overall goals of PR commits
- [x] Templates been checked with relevant user-story

## PR Todo's

Please refer to PR [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-submit-a-pull-request)

- [x] Specified reviewers (even if it's yourself)
- [x] Specified assignees (those who'll merge)
- [x] Specified labels
- [x] Specified milestone

## PR review

Please refer to PR review [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-review-a-pull_request)

- [ ] Code syntax formatting checked
- [ ] Debug messages are absent

## PR merge

Select only one:

- [ ] Should be merged?
- [x] Should be rebased?
- [ ] Should be squashed?

Please refer to PR merge [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-merge-a-pull_request)

### CODE REVIEW

File: arc_application/contact_centre.py [hunk 1]
```diff
@@ -58,9 +76,18 @@ def search(request):
...
                     context['error_text'] = 'Check that you have the correct details and spelling.'
+
+                    # Display all applications on search page
+                    results = Application.objects.filter()
+
+                    if results is not None and len(results) > 0:
+                        data = format_data(results)
+                        context['empty'] = False
+                        context['app'] = data
+
```
Comment (kingainformed) [inline arc_application/contact_centre.py]: Can this code be written pre-if statement?

**Comment by ricaduchateau** [comment]
Hi @kingainformed , I've changed the base branch to release/v.1.2.0 (created off of release/v1.1.0)

File: arc_application/contact_centre.py (hunk 1 above)
Comment (kingainformed) [reply arc_application/contact_centre.py]: could you have a default results set that displays on page load, if there are no records and if you make an invalid search request?

Comment (ricaduchateau) [reply arc_application/contact_centre.py]: Hi @kingainformed, made a new set of changes. The default results set is now created outside of the if GET/POST statement and I've tried to remove redundant code.

File: arc_application/contact_centre.py [hunk 2]
```diff
@@ -28,8 +28,17 @@ def search(request):
 
     if (cc_user or arc_user) and request.user.is_authenticated():
 
+        # Display all applications on search page
+        results = Application.objects.filter()
+
+        if results is not None and len(results) > 0:
+            data = format_data(results)
+            context['empty'] = False
```
[the remark this answers is not in the document]
Comment (ricaduchateau) [reply arc_application/contact_centre.py]: Hi @kingainformed, context['empty'] = False is to display the results table (set to True by default). context['empty_error'] displays the error summary at the top when set to True; in this case the error summary is not needed

[the remark this answers is not in the document]
Comment (kingainformed) [reply arc_application/contact_centre.py]: Cool!

**Pull request merged**